### PR TITLE
Add form element to add organ

### PR DESCRIPTION
**Pull request by jacobwindsor**
Related to [150](https://app.zenhub.com/workspaces/dcp-ingest-product-development-5f71ca62a3cb47326bdc1b5c/issues/ebi-ait/dcp-ingest-central/150)

### CODE REVIEW

File: client/src/app/project-registration/project-id/project-id.component.ts [hunk 1]
```diff
@@ -197,6 +224,15 @@ export class ProjectIdComponent implements OnInit {
...
+  }
+
+  private onOrganChange(val: any) {
+    const organ = this.metadataFormService.cleanFormData(val);
+    if (organ && organ.length > 0) {
+      this.organ = ProjectIdComponent.sanitiseOrgan(organ[0]['ontology_label']);
+    } else {
+      this.organ = '';
+    }
     this.generateProjectId();
```
Comment (aaclan-ebi) [inline client/src/app/project-registration/project-id/project-id.component.ts:236]: Why did you move the generateProjectId() to onOrganChange()? Is the organ value being used in the project id?

Comment (aaclan-ebi) [reply client/src/app/project-registration/project-id/project-id.component.ts:236]: i think the project id should still generated every time technology value changes.

**Review by aaclan-ebi — APPROVED** [review]
It looks good. I just want to note though, that because of the changes in the state tracker (allowing Exported -> Valid when a metadata is set back to draft). In the UI, any exported submissions whose project is updated and set back to Draft will make the submission valid again. It will be fixed if we check that only the project metadata content is updated before we set back to draft (... could deep compare the old project json vs the new project json) But yeah that's still an issue atm. You might set the submission back valid in prod if Marion / wranglers started to update this new field in the projects in prod.

File: client/src/app/project-registration/project-id/project-id.component.ts (hunk 1 above)
Comment (jacobwindsor) [reply client/src/app/project-registration/project-id/project-id.component.ts]: This was a mistake, thanks. Fixed now

**Comment by jacobwindsor** [comment]
Thanks @aaclan-ebi. That's a good point. @mshadbolt something to consider

**Comment by mshadbolt** [comment]
I don't think I understand what that means or what the implications are @jacobwindsor @aaclan-ebi

**Comment by mshadbolt** [comment]
I also didn't think the state should change on the project if we aren't editing the content of the metadata that isnt getting exported

**Comment by jacobwindsor** [comment]
@mshadbolt if you update the organ(s) of a project the submissions for that project will go from exported to valid. Obviously, there's no need to reexport them since we've just added a property that doesn't exist in the current schema

**Comment by aaclan-ebi** [comment]
That problem shouldn't happen in staging and prod now. We'll fix it in dev before promoting again. @mshadbolt you could go ahead update the new organ field for the projects is staging or prod (not sure if it's in prod now).

**Comment by aaclan-ebi** [comment]
And Thanks for filing a ticket for the issue ebi-ait/dcp-ingest-central#154 @jacobwindsor !
